protocol/block/mining_api_test.go

```diff
@@ -90,6 +89,28 @@ func TestMiningAPI_MiningStop(t *testing.T) {
 	assert.False(nd.IsMining())
 }
 
+func TestMiningAPI_MiningTogether(t *testing.T) {
+	tf.UnitTest(t)
+
+	assert := ast.New(t)
+	require := req.New(t)
```

Comment:
Blocking: this used to be the way we set up testing but we recently moved to calling the top level package functions directly without setting up these types.  `assert.True(nd.IsMining()` becomes `assert.True(t, nd.IsMining())` calling the package instead of these types.

Reply:
I copy from other test case in the mining_api_test.go. Need all test case update?